app/controllers/steps/income/answers_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Steps
+  module Income
+    class AnswersController < Steps::IncomeStepController
+      include Steps::NoOpAdvanceStep
```

Comment:
What is `NoOp` ?

Reply:
@RoseSAK For some forms we dont need any DB operation. Income CYAa page is one of them. Just display the answers and move on